script/validate_json.rb

```diff
@@ -20,7 +20,7 @@ module ExitCodes
     UNEXPECTED_NOTES = 12          # Unexpected notes key for translation
 end
 
-SupportedDifficulties = ["easy", "medium", "hard", "impossible"]
+SupportedDifficulties = ["easy", "medium", "hard", "only-required", "impossible"]
```

Comment:
```suggestion
SupportedDifficulties = ["easy", "medium", "hard", "only_required", "impossible"]
```

All other two word keys in our JSON use `_` as the separator.